### DISCUSSION TITLE
What's the performance of Byve?

### DISCUSSION THREAD

**Discussion by rempas** [bevyengine/bevy#2844] [Q&A]
Hi! I just wanted to ask about the performance of Byve. I see in the "README" that the "wgpu" is used which is something I never heard of. So compared to Vulkan, what's the performance? This reason I haven't got in Game development yet is because I'm waiting for Godot 4 that will include Vulkan so performance really matters to me.

**Comment by alterae** [comment]
You might find some answers in [wgpu's readme](https://github.com/gfx-rs/wgpu).

**Reply by rempas** [reply]
Thanks! Unfortunately I wasn't able to find anything specific to performance but from my understanding, this runs in top of Vulkan, OpenGL etc. so I suppose it just adds a small overload so the performance will be similar
